### ISSUE TITLE
Contract object missing?

### ISSUE THREAD

**Issue by iboxgithub** [alchemyplatform/alchemy-sdk-js#130]
Hi,

I am not able to instantiate a Contract object the same way I would do with ethers.js (with the contract ABI to then be able to call methods) and can't find a doc explaining how to do it. 
If you could point me in the right direction that would be great.

I see it here: https://github.com/alchemyplatform/alchemy-sdk-js/blob/10e96ee8b01aff0058158abd580afb38efb151f7/test/unit/contract.test.ts 
but if I do:
`const { Alchemy, Network, Wallet, Utils, Contract } = require("alchemy-sdk");`
Contract is the only object not recognized, and I can't add methods 

thanks

**Comment by thebrianchen** [comment]
@iboxgithub Thanks for writing in! We just added the Contract object in #98, and we're going to cut a new release containing this addition soon. Closing the issue for now, but feel free to ask any other questions you may have!

**Comment by iboxgithub** [comment]
wait so how can I use Contracts with alchemy-sdk @thebrianchen ???

**Comment by thebrianchen** [comment]
We just published `v2.1.0`. If you run `npm install alchemy-sdk@latest` it should work!

**Comment by kkharji** [comment]
Hello @thebrianchen ,

Just spent few hours trying to get the contract methods to appear. Is it support like with ethers?

```
    const provider = await this.alchemy.config.getProvider();
    const Address = "0x55eA9869eCEfd4487651285caweou23234289af8";
    const contract = new Contract(
      Address,
      json.abi,
      provider,
    );

    console.log(contract);
    collateral.emit("version", []);
```

I used to do `web3.eth.Contract(json.abi, Address)` and get methods key with all contract methods. Am I missing something, thanks
